data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -211,7 +211,11 @@ podman run \
 	--volume /etc/kubernetes:/etc/kubernetes:z \
 	--network=host \
 	"${CLUSTER_BOOTSTRAP_IMAGE}" \
-	start --asset-dir=/assets --required-pods openshift-kube-apiserver/openshift-kube-apiserver,openshift-kube-scheduler/openshift-kube-scheduler,openshift-kube-controller-manager/openshift-kube-controller-manager,openshift-cluster-version/cluster-version-operator
+	start \
+		--asset-dir=/assets \
+		--strict \
+		--tear-down-event run-forever \
```

Comment:
I am not convinced this is a good idea. We do not solve the load balancer problem, we only push it from ELB related to be kube-proxy related. All clients using in-cluster connection using the service IP will fall over (in the best case when the bootstrap machine is terminated). In the worst case, they will have pending watch TCP connection to a machine which is not there anymore. 

@squeed can you comment about that? How do TCP connections to service IPs behave if the target machine disappears?